### ISSUE TITLE
Remove / Disable Post Elements

### ISSUE THREAD

**Issue by bfintal** [gambitph/Page-Builder-Sandwich#132]
Originally from: https://wordpress.org/support/topic/remove-disable-post-elements by @abaney

Is it possible to remove / disable "post elements"? For example, I'd like to remove the capability for someone to add everything except for the 3 "Media" elements and the "Toggle" element. Some PHP to add to my theme's functions.php file would be the best.

If you're plugin can do this, then I'm sold! You've done an excellent job.

**Comment by bfintal** [comment]
@abaney Right now it's set so that anyone who has access to TinyMCE can add them. It's entirely possible though to add some kind of filter to disable sandwich. Would that work?

**Comment by abaney** [comment]
Actually, I decided that I'd prefer to use WordPress' inbuilt way of adding video (self-hosted or externally hosted) instead. To fix this issue, I added some CSS to the admin that will hide the "Add Post Elements" button.

Basically, the only thing I use is the column builder, which is amazing!

Still, it would probably be useful to have this option (described originally here) available in case something changes in the future, or in case someone else finds it useful.
